src/utils/errors.rs

```diff
@@ -15,14 +20,14 @@ use thiserror::Error;
 
 pub type ApiResult<T, E = ApiError> = Result<T, E>;
 
-#[derive(Error, Debug, PartialEq)]
+#[derive(Error, Debug, PartialEq, schemars::JsonSchema)]
```

Comment:
Why do we reference the `JsonSchema` here differently then in all other places? (`schemars::JsonSchema` vs `JsonSchema`)